### DISCUSSION TITLE
Eye tracking

### DISCUSSION THREAD

**Discussion by billybobilly** [vladmandic/face-api#177] [Q&A] [answered]
Has anyone implemented eye tracking to direct mouse pointers using face-api?  I see eyes are identified, but not pupils.

**Comment by vladmandic** [comment, marked as answer]
take a look at my much newer library, [Human](https://github.com/vladmandic/human)
precision of eye tracking is far from perfect - after all, it has to be able to execute in browser and eye iris is very small compared to overall input picture, but it does work.

**Reply by billybobilly** [reply]
Thank you for pointing me there.
